person.rb

```diff
@@ -1,4 +1,5 @@
-class Person
+require_relative 'nameable_interface'
```

Comment:
- [ ] Hello and nice job on refactoring the codes but the **require_relative** is not the correct way to call the function please refactor it to `attr_accessor :nameable`